vernac/declare.ml

```diff
@@ -97,8 +97,8 @@ end
 
 (** Declaration of constants and parameters *)
 
-type 'a proof_entry = {
-  proof_entry_body   : 'a Opaques.const_entry_body;
+type 'a pproof_entry = {
```

Comment:
What does `pproof` mean? Pure proof entry?

Reply:
Polymorphic. That's a naming scheme we already use in several other places.